rllib/algorithms/algorithm_config.py

```diff
@@ -792,6 +815,11 @@ def validate(self) -> None:
                 # compatibility for now. User only needs to set num_rollout_workers.
                 self.input_config["parallelism"] = self.num_rollout_workers or 1
 
+        # resolve rl_module class
+        if self._enable_rl_module_api and self.rl_module_class is None:
+            rl_module_class = self.get_rl_module_class(framework_str=self.framework_str)
```

Comment:
self.get_rl_module_class -> self.get_default_rl_module_class fits better here. If user does not override to a specific class, use this method to get the default class (analogous to how we currently do `get_default_policy_class` in `Algorithms`).

Reply:
yep. fixed.